WebPhapp/PharmaChain/update_prescription.js

```diff
@@ -28,8 +35,7 @@ async function update( drugChainIndex, dispenserID,drugQuantity, fulfillmentDate
 
     // Set up prescription data to be sent.
     Patient.options.address = fs.readFileSync("./patient_contract_address.txt").toString('ascii');
-    let transaction = await Patient.methods.updatePrescription(drugChainIndex, dispenserID,drugQuantity, 
-                                                        fulfillmentDate, daysValid, isCancelled, cancelDate);
+    let transaction = await Patient.methods.updatePrescription(drugChainIndex, dispenserID, drugQuantity, daysValid, refillsLeft);
```

Comment:
@jacobkrantz  are you okay with awaits being used here? They are so much easier to use than promises, in my opinion.